mitxgraders/comparers/linear_comparer.py

```diff
@@ -18,12 +18,12 @@ def get_linear_fit_error(x, y):
     Zero error in a linear relationship:
     >>> x = np.array([2, 5, 8])
     >>> result = get_linear_fit_error(x, 2*x + 1)
-    >>> round(result, 8)
+    >>> round(result, 6)
     0.0
     """
     A = np.vstack([x, np.ones(len(x))]).T
     _, residuals, _, _ = np.linalg.lstsq(A, y, rcond=-1)
-    return np.sqrt(residuals)
+    return np.sqrt(np.asscalar(residuals))
```

Comment:
@jolyonb Actually, I'm changing this to `np.asscalar`... residuals is always a 1-item array since we flatten the student input. 

BTW: `residuals` is actually the squared residuals, but I called it residuals b/c that's what numpy does https://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.lstsq.html

Reply:
Ok, `asscalar` makes more sense to me!